api-inference-community/requirements.txt

```diff
@@ -5,3 +5,6 @@ parameterized==0.8.1
 pillow==8.2.0
 huggingface_hub>=0.0.12
 datasets==1.10
+pytorchvideo==0.1.3
+torch==1.9.0
```

Comment:
This `torch` dependency is the exact reason why I wrote [this package](https://github.com/nateraw/encoded-video). With my version, we can forego the need for `torch`. Can include that logic here.

What do you think @Narsil?

Reply:
We are in agreement here! We clearly need an agnostic solution.